process-manager.js

```diff
@@ -88,38 +93,46 @@ class ProcessManager {
 	release(process) {
 		process.release();
 	}
+
 	send() {
 		if (!this.processes.length) {
 			return Promise.resolve(this.receive.apply(this, arguments));
 		}
 
+		// Prevents the arguments object from leaking.
+		let args = [];
+		for (let i = 0; i < arguments.length; i++) {
+			args.push(arguments[i]);
+		}
```

Comment:
> passing any direct reference to the arguments object to any other function

But this wasn't being done :|

Except for ``return Promise.resolve(this.receive.apply(this, arguments));``, which should be safe for being ``Function#apply``, and you aren't removing anyway...